api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -59,6 +59,14 @@ def bq_ingest(tier, source_project, dataset_name, table_match_filter="", table_s
   # bq init --delete_credentials as recommended in the output.
   # TODO(RW-3768): Find a better solution for Google credentials in docker.
 
+  # validate the CDR's tier label against the user-supplied tier, as a safety check
+
+  cdr_metadata = JSON.parse(`bq show --format=json "#{source_fq_dataset}"`)
```

Comment:
for consistency with the rest of the file, I would probably use `common.capture_stdout %{bq show --format=json #{fq_dataset}}` rather than backticks, e.g. see line 92

Reply:
ooh thanks